tasks/aoc.ts

```diff
@@ -1,9 +1,16 @@
 import {Client, TextChannel} from "discord.js";
 import {TaskExecutor} from "../types";
-import {embedLeaderboard} from "../lib/aocleaderboardparser"
+import {embedLeaderboard} from "../lib/aocleaderboardparser";
 import {get} from "../lib/configmanager";
 import {DateTime} from "luxon";
 
+/**
+ * TaskExecutor that sends the current standings for the AoC competition.
```

Comment:
Should we also make the documentation here uniform with the others in their wording at the start?